### PR TITLE
167 scrapy test

### PR DESCRIPTION
**Pull request by bylsmad**
https://github.com/moinwiki/moin/commit/ee6491c055696e1731efd8f3c5e31203262a6e8c makes adjustments for #1413 note that that PR resolved most known issues and introduced one new one

also note that after #1413 the Home link on help-en/html is pointing to help-en/Home is that correct?  was it meant to point to /Home ?

I was not able to reproduce the test failure referenced in the TODO in #1413 in test_modify_item, so that block was removed

### CODE REVIEW

File: src/moin/cli/_tests/test_modify_item.py [hunk 1]
```diff
@@ -50,10 +52,6 @@ def test_dump_help(load_help):
         # data_file_names = set([p.name for p in help_subdir_path.glob('*.data')])
         # assert expected_data_file_names == data_file_names
         for data_file_name in expected_data_file_names:
-            # TODO remove next 3 lines - adding new help-en items causes test failure
```
Comment (bylsmad) [inline src/moin/cli/_tests/test_modify_item.py:53]: could not reproduce this test failure

File: src/moin/help/welcome/users-Home.data [hunk 2]
```diff
@@ -0,0 +1 @@
+users/Home page
```
Comment (bylsmad) [inline src/moin/help/welcome/users-Home.data:1]: could we replace this with a wiki page describing what is the users namespace and what can be done here?  as a beginner moin user the users namespace is mysterious to me.

Comment (RogerHaase) [reply src/moin/help/welcome/users-Home.data:1]: yes, good idea

File: src/moin/cli/_tests/test_modify_item.py (hunk 1 above)
Comment (RogerHaase) [reply src/moin/cli/_tests/test_modify_item.py:53]: after merge and pull into master, the problem went away

File: src/moin/help/welcome/users-Home.data (hunk 2 above)
Comment (bylsmad) [reply src/moin/help/welcome/users-Home.data:1]: submitted this idea as #1416

**Pull request merged**